### DISCUSSION TITLE
Data-centric AI Chapter

### DISCUSSION THREAD

**Discussion by profvjreddi** [harvard-edge/cs249r_book#382] [Chapter Discussions]
Currently, the book focuses very much on pure machine learning systems engineering and really neglects the role of data in how we engineer the machine learning systems and this feels like a topic that would be good for the advanced section. While working on this, the data engineering chapter might need some improvement.

Welcoming thoughts, ideas, and suggestions on what this chapter should include.

**Comment by profvjreddi** [comment]
Some good food for thought around data from the EPOCHS folks regarding dataset scaling size

![CleanShot 2024-08-23 at 07 49 37@2x](https://github.com/user-attachments/assets/d037637d-5ec3-4f1f-9d56-b9e99d9f02a3)

**Comment by PursuitOfCodeKnowledge** [comment]
Your book is wounderfull!!! Just think about it!!! Time time and else time :))
